external/test-runtime/XUnit.Runtime.depproj

```diff
@@ -13,12 +13,13 @@
     <!-- Use xunit for arm packages because of not providing armel package in win10 -->
     <RuntimeIdentifier Condition="'$(ArchGroup)' == 'armel'">win10-arm</RuntimeIdentifier>
     <OutputPath>$(RuntimePath)</OutputPath>
```

Comment:
Ideally we just restore everything into the actual testhost folder and not spoil the RuntimePath but that's a too big change. For that we would need a new property that points to the actual runtime directory in the TestHostRootPath. 

Getting rid of the runtime and testhost folder duplication would be phenomenal.